draft-ietf-acme-dns-account-01.mkd

```diff
@@ -62,44 +63,140 @@ systems or environments to handle challenge-solving for a single domain.
 
 The `dns-01` challenge specified in section 8.4 of {{!RFC8555}} requires that
 ACME clients validate the domain under the `_acme-challenge` label for the
-`TXT` record. This unique label creates an impediment limiting the number of
-other entities domain validation can be delegated to.
-
-In multi-region deployments, where separate availability zones serve the same
-content, and dependencies across them are avoided, operators need a way to
-obtain a separate certificate per zone, for the same domain name. Similarly, in
-cases of zero-downtime migration, two different setups of the infrastructure
-may coexist for a long period of time, and both need access to valid
+`TXT` record. This label creates several limiations in domain validation.
+
+First, the `_acme-challenge` label does not specify if the authorization is
+intended for a specific host, a wildcard domain, or a domain and all of its
+subdomains. Consequently, when validating a specific DNS name with the DNS-01
```

Comment:
```suggestion
subdomains. Consequently, domain owners who may be delegating or provisioning authorization labels for a domain must concede control over the domain and all subdomains, violating the principle of least privilege.
```

I think something like this might replace a few of the sentences below in this paragraph. 

In particular, I think we should bring up the principle of least privilege. There are probably better ways to word it.